src/unset.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const pathToArray = require('./util/pathToArray')
+const isEmpty = require('./isEmpty')
+const isPlainObject = require('./isPlainObject')
+const get = require('./get')
+
+const _doUnset = (obj, path) => {
+  const level = path.length - 1
+  const key = path.shift()
+  if (obj[key] !== undefined) {
```

Comment:
what happens if obj[key] is null ... it won't be deleted and it should

Reply:
It actually does deletes fields whose values are null. I've updated the tests to verify this.